### PR TITLE
Feature/ros removal

### PR DESCRIPTION
**Pull request by PierreLucGriffin**

### CODE REVIEW

**Review by LucRivNor — DISMISSED** [review]

File: GodotProjects/main.gd [hunk 1]
```diff
@@ -53,6 +54,7 @@ func _on_options_menu_index_pressed(index):
 		1: _disconnection_to_scene_manager()
 
 func update_joints(joints):
+	#print("msg received")
```
Comment (UlricVilleneuveOmni) [inline GodotProjects/main.gd]: Remove dead code.

File: GodotProjects/main.gd [hunk 2]
```diff
@@ -61,7 +63,13 @@ func update_joints(joints):
 		var joint_name = joints["joints_name"][i]
 		var joint_value = joints["joints_value"][i]
 		#get_tree().get_root().get_node(joint_name).set_joint_value(joint_value)
-		get_node("..").get_node(joint_name).call_deferred("set_joint_value", joint_value)
+		
+		var names = split_node_path(joint_name)
+		
+		#print("name0:",names[0], "maadad:", names[1])
```
Comment (UlricVilleneuveOmni) [inline GodotProjects/main.gd]: Remove dead code.

File: GodotProjects/node_finder.gd [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+extends Node
+
+static func find_node_path(scene, str_start="", str_end=""):
+	for child in scene.get_children():
+		var node = _recursive_get_name(child, str_start, str_end)
+		if node != null:
+			return node
+
+static func _recursive_get_name(node, str_start="", str_end="", current_name = ""):
+# we need to do this because otherwise the child with no owners
```
Comment (UlricVilleneuveOmni) [inline GodotProjects/node_finder.gd]: This comment should be removed, it does not make sense anymore

File: modules/aos/scene/AosScene.cpp [hunk 4]
```diff
@@ -584,12 +584,12 @@ namespace aos
...
         {
-            //omni::document::document<Omni::Geometry::Mesh::SimpleMesh, Omni::Geometry::Mesh::ply>* doc;
-            //doc = new omni::document::document<Omni::Geometry::Mesh::SimpleMesh, Omni::Geometry::Mesh::ply>(store_key);
+            omni::document::document<Omni::Geometry::Mesh::SimpleMesh, Omni::Geometry::Mesh::ply>* doc;
+            doc = new omni::document::document<Omni::Geometry::Mesh::SimpleMesh, Omni::Geometry::Mesh::ply>(store_key);
             try
             {
-                mesh = GodotResolveMesh<Omni::Geometry::Mesh::ply>(store_key, false); // false for not binary
-                //mesh = doc->resolve_object().get();
+                //mesh = GodotResolveMesh<Omni::Geometry::Mesh::ply>(store_key, false); // false for not binary
```
Comment (UlricVilleneuveOmni) [inline modules/aos/scene/AosScene.cpp]: Remove dead code, and the method GodotResolveMesh can be removed as well.

File: modules/scene_manager/include/google/protobuf/any.h [hunk 5]
```diff
@@ -0,0 +1,149 @@
+// Protocol Buffers - Google's data interchange format
```
Comment (UlricVilleneuveOmni) [inline modules/scene_manager/include/google/protobuf/any.h:1]: Do we need all these protobuf .h files?

**Review by UlricVilleneuveOmni — CHANGES_REQUESTED** [review]

File: modules/scene_manager/include/google/protobuf/any.h (hunk 5 above)
Comment (PierreLucGriffin) [reply modules/scene_manager/include/google/protobuf/any.h:1]: @olivierp99 Voir commentaire d'Ulric.

File: modules/aos/scene/AosScene.cpp (hunk 4 above)
Comment (PierreLucGriffin) [reply modules/aos/scene/AosScene.cpp]: GodotResolveMesh can't be removed. It's still used.

**Comment by PierreLucGriffin** [comment]
All work has been moved to OmniGodot repo.

**Pull request closed without merging**